Change templates to be more like in old Debian version,
esp. use <FORM> instead of <ISINDEX>

Index: man-1.6g/man2html/scripts/cgi-aux/man/man.aux
===================================================================
--- man-1.6g.orig/man2html/scripts/cgi-aux/man/man.aux	2011-01-09 17:09:35.000000000 +0100
+++ man-1.6g/man2html/scripts/cgi-aux/man/man.aux	2011-01-09 17:11:20.000000000 +0100
@@ -1,5 +1,7 @@
 Content-type: text/html
 
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+            "http://www.w3.org/TR/html40/loose.dtd">
 <HTML>
 <HEAD>
 <TITLE>Manual Pages - Main Contents</TITLE>
@@ -11,55 +13,51 @@
 <HR>
 
 <H2>Name and Section lookup</H2>
-<ISINDEX>
+<FORM method="GET" action="%cg/man2html">
+	<INPUT NAME="query" SIZE="40">
+	<INPUT TYPE="submit" VALUE="Search">
+</FORM>
 
-%lynx <i>Start a query</i> by typing an s.
 You can enter a program name, possibly preceded by the section,
 the directories to search (with -M) or a full name.
 For example:
 <UL>
    <LI><TT>find</TT>
    <LI><TT>1 find</TT>
-   <LI><TT>-M /usr/man:/usr/X11R6/man:/usr/lib/perl5/man find</TT>
-   <LI><TT>/local/gcc/man/man1/gperf.1</TT>
+   <LI><TT>-M /usr/share/man:/opt/man:/usr/local/share/man find</TT>
+   <LI><TT>/usr/share/man/man1/gperf.1</TT>
 </UL>
 
 <HR>
 
 <H2>Index of pages by name and description</H2>
 Sections:
-  <A HREF="%cg/manwhatis?%cg+1">1. User Commands</A>;
-  <A HREF="%cg/manwhatis?%cg+2">2. System Calls</A>;
-  <A HREF="%cg/manwhatis?%cg+3">3. Library Functions</A>;
-  <A HREF="%cg/manwhatis?%cg+4">4. Special Files</A>;
-  <A HREF="%cg/manwhatis?%cg+5">5. File Formats</A>;
-  <A HREF="%cg/manwhatis?%cg+6">6. Games</A>;
-  <A HREF="%cg/manwhatis?%cg+7">7. Miscellany</A>;
-  <A HREF="%cg/manwhatis?%cg+8">8. Administration and Privileged Commands</A>;
-  <A HREF="%cg/manwhatis?%cg+l">l. Postgresql Commands</A>;
-  <A HREF="%cg/manwhatis?%cg+n">n. Tcl Commands</A>.
+  <A HREF="%cg/manwhatis?1">1. User Commands</A>;
+  <A HREF="%cg/manwhatis?2">2. System Calls</A>;
+  <A HREF="%cg/manwhatis?3">3. Library Functions</A>;
+  <A HREF="%cg/manwhatis?4">4. Special Files</A>;
+  <A HREF="%cg/manwhatis?5">5. File Formats</A>;
+  <A HREF="%cg/manwhatis?6">6. Games</A>;
+  <A HREF="%cg/manwhatis?7">7. Miscellany</A>;
+  <A HREF="%cg/manwhatis?8">8. Administration and Privileged Commands</A>.
 <P>
 <HR>
 <H2>Index of pages by name only</H2>
-  <A HREF="%cg/mansec?%cg+1">1. User Commands</A>;
-  <A HREF="%cg/mansec?%cg+2">2. System Calls</A>;
-  <A HREF="%cg/mansec?%cg+3">3. Library Functions</A>;
-  <A HREF="%cg/mansec?%cg+4">4. Special Files</A>;
-  <A HREF="%cg/mansec?%cg+5">5. File Formats</A>;
-  <A HREF="%cg/mansec?%cg+6">6. Games</A>;
-  <A HREF="%cg/mansec?%cg+7">7. Miscellany</A>;
-  <A HREF="%cg/mansec?%cg+8">8. Administration and Privileged Commands</A>;
-  <A HREF="%cg/mansec?%cg+l">l. Postgresql Commands</A>;
-  <A HREF="%cg/mansec?%cg+n">n. Tcl Commands</A>;
-  <A HREF="%cg/mansec?%cg+all">All Sections</A>.
+  <A HREF="%cg/mansec?1">1. User Commands</A>;
+  <A HREF="%cg/mansec?2">2. System Calls</A>;
+  <A HREF="%cg/mansec?3">3. Library Functions</A>;
+  <A HREF="%cg/mansec?4">4. Special Files</A>;
+  <A HREF="%cg/mansec?5">5. File Formats</A>;
+  <A HREF="%cg/mansec?6">6. Games</A>;
+  <A HREF="%cg/mansec?7">7. Miscellany</A>;
+  <A HREF="%cg/mansec?8">8. Administration and Privileged Commands</A>;
+  <A HREF="%cg/mansec?all">All Sections</A>.
 <P>
 <hr>
 
 <H2>Manual Pages full text search</H2>
 
-<A HREF="%cg/mansearch">
-Search the full text of the Manual Pages.
-</A>
+<A HREF="%cg/mansearch">Search the full text of the Manual Pages</A>.
 
 <P>
 <HR>
@@ -73,3 +71,4 @@
 Federico Lucifredi</A>.
 </body>
 </HTML>
+<!-- vim: set ft=html: -->
Index: man-1.6g/man2html/scripts/cgi-aux/man/mansearch.aux
===================================================================
--- man-1.6g.orig/man2html/scripts/cgi-aux/man/mansearch.aux	2011-01-09 17:09:21.000000000 +0100
+++ man-1.6g/man2html/scripts/cgi-aux/man/mansearch.aux	2011-01-09 17:09:36.000000000 +0100
@@ -1,5 +1,7 @@
 Content-type: text/html
 
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+            "http://www.w3.org/TR/html40/loose.dtd">
 <HTML>
 <HEAD>
 <TITLE>Manual Pages - Text Search</TITLE>
@@ -11,39 +13,33 @@
 
 <A HREF="%cg/man2html">Return to Main Contents</A>
 <P>
-<ISINDEX>
-
-%lynx <i>Start a query</i> by typing an s.
-
+<FORM method="GET" action="%cg/mansearch">
+	<INPUT NAME="query" SIZE="40">
+	<INPUT TYPE="submit" VALUE="Search">
+</FORM>
 Examples:
 <UL>
-   <LI><TT>pwd;directory</TT> - Find both in the same man page.
-   <LI><TT>pwd,directory</TT> - Find either.
-   <LI><TT>pwd directory</TT> - Find the exact phrase.
-   <LI><TT>-F 1 pwd</TT> - Look in section 1 only.
-   <LI><TT>-B glipmse</TT> - Find the best match (allow for misspelling).
-   <LI><TT>-L 0:10 pwd</TT> - Only show the first 10 man pages matched.
+   <LI><TT>mouse and computer</TT> - Find both in the same man page.</LI>
+   <LI><TT>mouse or computer</TT> - Find either.</LI>
+   <LI><TT>mous*</TT> - Partial match.</LI>
+   <LI><TT>name=bourne*</tt> - Partial match in NAME-section.</LI>
 </UL>
 
 <HR>
 <P>
-The index uses the <I>Glimpse</I>
-(<A HREF="%cg/man2html?1+glimpse">glimpse(1)</A>)
-text indexing system.  Many of the glimpse
-options, regular expressions, and wildcards are valid:
-see the <A HREF="%cg/mansearchhelp">help page</A> for a summary.
+The index uses the
+<A HREF="http://homepage.mac.com/pauljlucas/software/swish/"><I>Swish++</I></A>
+indexing system. See <A HREF="%cg/man2html?query=search%2B%2B+1">search++(1)</A>
+and <A HREF="%cg/man2html?query=index%2B%2B+1">index++(1)</A>
+for detailed queries.
+Meta-Names: AUTHOR, BUGS, CAVEATS, DESCRIPTION,  DIAGNOSTICS, ENVIRONMENT,
+ERRORS, EXAMPLES, EXIT-STATUS, FILES, HISTORY, NAME, NOTES, OPTIONS,
+RETURN-VALUE, SEE-ALSO, SYNOPSIS, and WARNINGS
 <P>
 <HR>
-
-<P>
-Glimpse
-was developed by Udi Manber and Burra Gopal of the University of
-Arizona, and Sun Wu of the National Chung-Cheng University, Taiwan.
-You may contact the authors at <A HREF="mailto:glimpse@cs.arizona.edu">
-glimpse@cs.arizona.edu</A>
-<P>
 The Manual Page text search interface was created for Linux by
 <A HREF="mailto:michael@actrix.gen.nz">
 Michael Hamilton</A>.
 </BODY>
 </HTML>
+<!-- vim: set ft=html: -->
